src/components/scroller.js

```diff
@@ -575,7 +575,9 @@ define(['browser', 'layoutManager', 'dom', 'focusManager', 'ResizeObserver', 'sc
 		 */
         function normalizeWheelDelta(event) {
             // wheelDelta needed only for IE8-
-            scrolling.curDelta = ((o.horizontal ? event.deltaY || event.deltaX : event.deltaY) || -event.wheelDelta);
+            // JELLYFIN MOD: Only use deltaX for horizontal scroll instead of `event.deltaY || event.deltaX`
+            scrolling.curDelta = ((o.horizontal ? event.deltaX : event.deltaY) || -event.wheelDelta);
```

Comment:
Firefox 72.0.1 on mouse wheel reports
```
event.deltaX = 0;
event.deltaY = 3;
```
In result, `scrolling.curDelta = -event.wheelDelta <[undefined] = NaN`

I tried to leave as it was, replace `passive: true` with `capture: true` for `scrollHandler` and add `event.preventDefault()` in `scrollHandler` after this block
https://github.com/jellyfin/jellyfin-web/blob/5fb2c8bd82b3dae4ad629d38712f683935ad06cf/src/components/scroller.js#L600-L602
In this case, emby-scroller scrolls when hovered, window does not scroll (even scroller at very start/end). To scroll window, need to place cursor over empty place of page.

Reply:
I removed the fallback to `wheelDelta` since that was only present to support IE8 based on the comment. It is working well for me now on Firefox 73 and Chrome 80.